benchmarks/config/GPU/cuda.json

```diff
@@ -0,0 +1,71 @@
+[
+  {
+  "cpu_gemm": {
```

Comment:
Unnecessary to run CPU as a baseline for GPU, we can't really compare them 1:1 and we already have them in other runs, even quick ones.